quickcheck-dynamic/quickcheck-dynamic.cabal

```diff
@@ -89,6 +90,7 @@ test-suite quickcheck-dynamic-test
   main-is:        Spec.hs
   hs-source-dirs: test
   other-modules:
+    Spec.DynamicLogic.CounterModel
```

Comment:
This new module does not seem related with the purpose of this PR

Reply:
It's there to check that if you don't define an error type, you don't need to return an `Either` from `perform`.